src/components/MenuAction.tsx

```diff
@@ -53,18 +55,13 @@ const useStyles = makeStyles((theme) => ({
   }
 }));
 
-interface MenuActionItemsProps {
-  idx: number;
-  title: string;
-  isEnabled: boolean;
-}
 interface MenuActionProps {
-  items: Array<MenuActionItemsProps>;
-  menuIdx: number;
-  changeMenu: Dispatch<SetStateAction<number>>;
+  actions: Array<MenuActionItemsProps>;
```

Comment:
This is not wrong at all or a blocker. But in order to satisfy my OCD I believe this should be altered in order to be consistent with `src/contexts/GUIContextProvider.tsx` (line 27)

```suggestion
  actions: MenuActionItemsProps[];
```